configure.ac

```diff
@@ -301,6 +301,7 @@ AC_CHECK_FUNCS( futimes )
 AC_CHECK_FUNCS( wcslcpy lrand48_r killpg )
 AC_CHECK_FUNCS( backtrace_symbols getifaddrs )
 AC_CHECK_FUNCS( futimens clock_gettime )
+AC_CHECK_FUNCS( localtime_r )
```

Comment:
`AC_CHECK_FUNCS` defines a preprocessor symbol if the function is available, but there is no check for this in the code (and no alternative handling).